### ISSUE TITLE
Warn on missing fallthrough comment in switch

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1081]
```javascript
switch (c) {
case a:
  foo();
case b:
  bar();
}
```
This code should warn after `foo();` saying that a fallthrough comment is missing.

**Comment by yashmasani** [comment]
can i work on this task :D

**Comment by yashmasani** [comment]
hey @strager,
for this warning, 
1. do leading and lagging comments matter?
`switch (c) {
case a:
  foo();
  // fallthrough strictly
  // case b is for the edge case x
case b:
  bar();
}`

2. Also, do you expect quick lint to be strict on the 'fallthrough' comment i.e // Fallthrough, // fallThrough are still valid comments

**Comment by strager** [comment]
> 
    do leading and lagging comments matter?

**Comment by strager** [comment]
> 1. do leading and lagging comments matter? [...]
> 2. Also, do you expect quick lint to be strict on the 'fallthrough' comment [...]

Good questions. I think we should not warn if there is *any* comment. In other words, I think we should warn only if there is *no* comment. It doesn't matter what the comment says.
